knowledge_graphs/ast_utils.py

```
@@ -244,238 +233,296 @@
             return f"{value}[{slice_val}]"
-        elif class_name == 'Index':  # Python < 3.9
+        elif class_name == "Index":  # Python < 3.9
             return _ast_node_to_string(node.value)
-        elif class_name == 'List':
+        elif class_name == "List":
             elements = [_ast_node_to_string(e) for e in node.elts]
             return f"[{', '.join(elements)}]"
-        elif class_name == 'Tuple':
+        elif class_name == "Tuple":
             elements = [_ast_node_to_string(e) for e in node.elts]
             return f"({', '.join(elements)})"
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)